nimare/meta/utils.py

```diff
@@ -275,6 +275,57 @@ def generate_input_fn():
     return niis
 
 
+def unique_rows(ar):
+    """Remove repeated rows from a 2D array.
+
+    In particular, if given an array of coordinates of shape
+    (Npoints, Ndim), it will remove repeated points.
+
+    Parameters
+    ----------
+    ar : 2-D ndarray
+        The input array.
+
+    Returns
+    -------
+    ar_out : 2-D ndarray
+        A copy of the input array with repeated rows removed.
+
+    Raises
+    ------
+    ValueError : if `ar` is not two-dimensional.
+
+    Notes
+    -----
+    The function will generate a copy of `ar` if it is not
+    C-contiguous, which will negatively affect performance for large
+    input arrays.
```

Comment:
Can you add a reference to the scikit-image function? E.g., ```This is taken from skimage. See :func:`skimage.util.unique_rows`.``` We'll need to add skimage to the inter sphinx mapping in `docs/conf.py`.

Reply:
Does that work w/o adding skimage as a dependency?

Reply:
yep, I figured it out I think.